src/python/turicreate/test/test_sframe.py

```diff
@@ -201,6 +201,13 @@ def test_auto_parse_csv_with_bom(self):
             sf = SFrame.read_csv(csvfile.name, header=True)
             self.assertEqual(sf.dtype, [float, int, str])
             self.__test_equal(sf, df)
+    def test_read_json(self):
+        x='{"row 1":{"col 1":"a","col 2":"b"},"row 2":{"col 1":"c","col 2":"d"}}'
```

Comment:
So this is Panda's default JSON format which is different than what our SFrame expects. If you were to write this string to a file and try to read it with `tc.SFrame.read_json(...)`, you would get an error.

The expected JSON format should not be different based on weather a string literal is used or a filename is given.